wcfsetup/setup/db/install.sql

```diff
@@ -1958,6 +1958,17 @@ CREATE TABLE wcf1_user_rank (
 	hideTitle TINYINT(1) NOT NULL DEFAULT 0
 );
 
+DROP TABLE IF EXISTS wcf1_user_rank_content;
+CREATE TABLE wcf1_user_rank_content (
+    contentID INT(10) NOT NULL AUTO_INCREMENT,
+	rankID INT NOT NULL,
+	languageID INT,
+	title VARCHAR(255) NOT NULL,
+
+	PRIMARY KEY(contentID),
+	UNIQUE KEY id (rankID, languageID)
```

Comment:
This does not work as expected, because as far as MySQL is concerned, `NULL` is not identical with `NULL`. It will happily let you insert `(1, null), (1, null), (1, null)` because those are three distinct tuples.

This is the same with `NaN` in JavaScript if that helps to memorize it.